pkg/converter/options.go

```diff
@@ -33,6 +34,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	}
 	fs.StringVar(&o.context, flagContext, "", "The name of the kubeconfig context to use")
 	fs.StringVar(&o.azureConfigDir, flagAzureConfigDir, "", "Azure CLI config path")
+	fs.StringVar(&o.execInstallHint, flagExecInstallHint, "", "The install hint for the kubelogin command when converting to exec format.")
```

Comment:
actually, I didn't mean to add an option for the hint. What i meant is, when the kubeconfig to be converted is already of exec format and has install hint, we should preserve it during conversion. For instance
```
az aks get-credentials... 
# the downloaded kubeconfig is in exec format and has aks specific install hint
kubelogin convert-kubeconfig -l msi
# the output should keep the same hint as before
```

Reply:
Makes sense. Reverted that change.